src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,112 @@
+package duke;
+
+import duke.exceptions.InvalidCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
+    private static final String DIRECTORY_NAME = "data";
+    private static final String FILE_NAME = "duke.txt";
+    private static final String DONE = "1";
+
+    public static void loadFile(TaskList taskList) {
+        try {
+            createDirectory();
+            createFile();
+            loadTasks(taskList);
+        } catch (IOException | SecurityException | InvalidCommandException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void saveFile(TaskList taskList) {
+        try {
+            createDirectory();
+            createFile();
+            saveTasks(taskList);
```

Comment:
Again, good use of [SLAP](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here.